.github/workflows/go.yml

```diff
@@ -24,6 +24,6 @@ jobs:
       - name: Build SDK project
         run: go build -p=1 -gcflags=all=-c=1
         working-directory: ${{ env.relativePath }}
-      - name: Run unit tests
-        run: go test -p=1 -gcflags=all=-c=1
-        working-directory: ${{ env.relativePath }}
+#      - name: Run unit tests
```

Comment:
?

Reply:
Tests runs were running for too long so I reverted it commenting it out as they were.

Reply:
fair, and we're already testing the same code in v1 anyway. Resolving.